### DISCUSSION TITLE
Custom krb5.keytab in /etc on clients

### DISCUSSION THREAD

**Discussion by macroendrix** [ltsp/ltsp#709] [General] [answered]
Is there a way to set custom krb5.keytab file only on clients? The purpose is to mount NFS with Kerberos enable on clients, but the default krb5.keytab is copied from server and only contain the HOST entries from server inself. Thanks in advanced.

**Comment by alkisg** [comment, marked as answer]
You can put them e.g. in /etc/ltsp/01:02:03:04:05:06.keytab and then copy them wherever they belong with:

POST_INIT_COPY_KEYTAB="cp -a /etc/ltsp/$MAC_ADDRESS.keytab /destination/path"

**Reply by macroendrix** [reply]
Thanks!
